trillian/ctfe/instance.go

```diff
@@ -93,6 +94,17 @@ type InstanceOptions struct {
 	// a boolean to indicate whether the conversion succeeded.
 	ErrorMapper func(error) (int, bool)
 	RequestLog  RequestLog
+	// RemoteUser returns a string representing the originating host for the
+	// given request. This string will be used as a User quota key.
```

Comment:
(assuming per-log config)
"will be used as a User quota key for any log that has per-user charging configured"

Reply:
Punting that for now